test/mockupdb/operations.py

```diff
@@ -14,15 +14,14 @@
 
 from collections import namedtuple
 
-from mockupdb import *
-from mockupdb import OpMsgReply
+from mockupdb import OpMsgReply, OpMsg, OpReply  # type: ignore
```

Comment:
What if we use the mypy.ini file to ignore all mockupdb imports instead:
```
[mypy-mockupdb]
ignore_missing_imports = True
```

Reply:
Done